README.md

```diff
@@ -1,56 +1,112 @@
 
----
+[⬆ Back to Top](#toc)
 
-[![NPM](https://img.shields.io/npm/v/@appbaseio/react-searchbox.svg)](https://www.npmjs.com/package/@appbaseio/react-searchbox) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+## 2. Features
 
-## Install
+- Design search experiences with best practices
+- Customize your components at will
+- Follow React principles
 
-```bash
-npm install @appbaseio/react-searchbox
-# or
-yarn add @appbaseio/react-searchbox
-```
+[⬆ Back to Top](#toc)
 
-## Usage
+## 3. Usage
 
 ### Basic Usage
 
 ```js
 <SearchBox
-	app="good-books-ds"
-	credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
-	dataField={['original_title', 'original_title.search']}
+  app="good-books-ds"
+  credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
+  dataField={['original_title', 'original_title.search']}
 />
 ```
 
 ### Usage With All Props
 
 ```js
 <SearchBox
-	app="good-books-ds"
-	credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
-	dataField={[
-		{ field: 'original_title', weight: 1 },
-		{ field: 'original_title.search', weight: 3 },
-	]}
-	title="Search"
-	defaultValue="Songwriting"
-	placeholder="Search for books"
-	autosuggest={true}
-	defaultSuggestions={[
-		{ label: 'Songwriting', value: 'Songwriting' },
-		{ label: 'Musicians', value: 'Musicians' },
-	]}
-	highlight={true}
-	highlightField="group_city"
-	queryFormat="or"
-	fuzziness="AUTO"
-	showClear
-	showVoiceSearch
+  app="good-books-ds"
+  credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
+  dataField={[
+    { field: 'original_title', weight: 1 },
+    { field: 'original_title.search', weight: 3 }
+  ]}
+  title="Search"
+  defaultValue="Songwriting"
+  placeholder="Search for books"
+  autosuggest={true}
+  defaultSuggestions={[
+    { label: 'Songwriting', value: 'Songwriting' },
+    { label: 'Musicians', value: 'Musicians' }
+  ]}
+  highlight={true}
+  highlightField="group_city"
+  queryFormat="or"
+  fuzziness="AUTO"
+  showClear
+  showVoiceSearch
 />
 ```
+
+[⬆ Back to Top](#toc)
+
+## 4. Installation
+
+```bash
+npm install @appbaseio/react-searchbox
+# or
+yarn add @appbaseio/react-searchbox
+```
+
+[⬆ Back to Top](#toc)
+
+## 5. Docs Manual
+
+The official docs for the library are at [docs/react-searchbox](https://docs.appbase.io/docs/reactivesearch/react-searchbox/quickstart/)
+
+[⬆ Back to Top](#toc)
+
+## 6. Contributing
+
+Please check the [contribution guide](./CONTRIBUTING.md)
+
+[⬆ Back to Top](#toc)
+
+## 7. Other Projects You Might Like
+
+- [**dejavu**](https://github.com/appbaseio/dejavu) allows viewing raw data within an appbase.io (or Elasticsearch) app. **Soon to be released feature:** An ability to import custom data from CSV and JSON files, along with a guided walkthrough on applying data mappings.
```

Comment:
@ShahAnuj2610 It seems you copied this from ReactiveSearch. Obviously, now that we're in Searchbox, the top relevant project would be ReactiveSearch itself.

I would remove ReactiveSearch Dashboard from here as we are deprecating it and update the link for ReactiveMaps (as it's a module inside of ReactiveSearch itself now).

We should also add a link to Arc (2nd in order) as it can work out of the box with Searchbox + ReactiveSearch libraries to capture the search + click analytics recorded here.